scripts/yosys/logic_equiv_check.tcl

```diff
@@ -13,110 +13,64 @@
 # limitations under the License.
 yosys -import
 
-if { [info exists ::env(FP_WELLTAP_CELL)] && $::env(FP_WELLTAP_CELL) ne ""} {
-	set well_tap_cell "$::env(FP_WELLTAP_CELL)"
+proc exclude_fills {args} {
+	set fills "$::env(CELL_PAD_EXCLUDE)"
```

Comment:
this is sort of a workaround

Reply:
The LEC script is broken as hell anyway, this represents a net delta of 0. We can revisit this later.

Reply:
Regardless, its a workaround. I checked the pdk vars and there are three variables than can be used here but they don't fully capture all the tap cells. Since resolving this requires some effort with the pdk I am going to approve the pull request but at the same time the limitation needs to be flagged